examples/Python/src/lib/WebServer.lf

```diff
@@ -0,0 +1,84 @@
+/**
+ * @file
+ * @author Shulu Li
+ * @brief Reactor for handling HTTP requests in Python.
+ */
+target Python {
+  keepalive: true
+}
+
+/**
+ * @brief A reactor that starts a FastAPI server to handle HTTP requests.
+ *
+ * The `port` parameter is the port number on which the server listens for requests.
+ *
+ * The `endpoint` parameter is the endpoint at which the server listens for requests.
+ *
+ * The `request` output is a tuple of two values: the request ID and the request data.
```

Comment:
Can you clarify what is the request data?  I assume this is the body of the POST, which is normally JSON (but I guess it can be any string?).